internal/git/git.go

```diff
@@ -238,6 +238,21 @@ func GetCurrentCommit(dir string) (string, error) {
 	return strings.TrimSpace(string(output)), nil
 }
 
+func Fetch(dir string) error {
+	var buffer bytes.Buffer
+
+	cmd := exec.Command("git", "fetch")
```

Comment:
Technically you copied my code, but I just realized this is the same as doing `CombinedOutput` instead of `Run`.
Not a requested changed, because there are more than 1 road to rome, but yeah. Brain fart!

Reply:
Yeap, indeed! I just fixed both funcs, please double-check! 🙏

Also made me realize that with `cmd.Dir = dir` we can avoid all the `"-C", dir` everywhere in that file. Not that it matters that much, but just makes me wonder why I didn't go with that approach in the first place(!) 🤷